spec/services/blacklight/search_service_spec.rb

```diff
@@ -485,4 +486,10 @@
       expect(service.opensearch_response.last).to match_array %w[A B C]
     end
   end
+
+  describe '#context' do
+    it 'has a context attribute' do
+      expect(subject.context).to eq context
```

Comment:
Shouldn't `subject.context == { context: context }` ?

Reply:
I've updated the way it's getting stubbed so this should be fine.